homework/CSharpOOP3/Ball.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP3
+{
+    internal class Ball : Shape
+    {
+        private string _name;
+        private double _radius;
+
+        public Ball(string name, double radius) : base(name)
+        {
+            _name = name;
+            _radius = radius;
+        }
+
+        public override double GetVolume()
+        {
+            double ballVolume = 1.33 * Math.PI * _radius * _radius * _radius;
```

Comment:
Read about Math.Round and Math.Pow.
Because It wolf better to use Math.Pow to raise a number to a power.
And try to avoid magic numbers like 1.33. I would recommend you to use full formula. ((4 * Math.PI * Math.Pow(Radius, 3)) / 3)